src/main/resources/endpoint.properties

```diff
@@ -15,4 +15,4 @@ _endpoints_services_api=http://localhost:2233/api
 _token=test1/dev/veritoneuser
 
 # Endpoint specific properties
-_endpoint_config={"environment":"stage", "clientId":"id", "clientSecret":"secret"}
+_endpoint_config={"environment":"stage", "region":"us-1", "clientId":"37d8b00c-9547-4b92-b6b6-f037c280b3c8", "clientSecret":"9kWCQoSyXrlzFfgPrE3VR80DYwr4sdhPWcOt7kfpFsxUvmbL3c-jJw", "redirectUri": "http://test1.localhost:8000/dev/endpoints/veritoneuser/authCallback"}
```

Comment:
I would delete the default configurations, and in case of using local, I would create an endpoint_local.properties file and add it to the gitignore.